addon/components/value-viewer.js

```diff
@@ -46,4 +46,12 @@ export default Component.extend({
 
     return count;
   }),
+
+  indentation: computed("depth", function() {
+    return "".padStart((this.get('depth') + 1) * 2 , " ");
```

Comment:
This would make more sense (imo) to specify 2-spaces as the pad string, and pad it `depth+1` times, rather than padding twice as many times of a 1-space string...
```suggestion
    let twoSpaces = "  ";
    return "".padStart((this.get('depth') + 1) , twoSpaces);
```